mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -96,6 +96,25 @@ def TilingInterface : OpInterface<"TilingInterface"> {
           return failure();
         }]
       >,
+      InterfaceMethod<
+        /*desc=*/[{
+          Method to return the position of iteration domain tile computed by the
+          tiled operation.
+        }],
+        /*retType=*/"LogicalResult",
```

Comment:
Nit: prefer globally-qualified types in .td, `::mlir::LogicalResult`. Generated code may be #included in another namespace.

Reply:
nice catch, done